pkg/models/order/order.go

```diff
@@ -48,22 +49,7 @@ type Cancel Order
 // FromRaw takes the raw list of values as returned from the websocket
 // service and tries to convert it into an Order.
 func FromRaw(raw []interface{}) (o *Order, err error) {
-	if len(raw) == 12 {
-		o = &Order{
-			ID:         convert.I64ValOrZero(raw[0]),
-			Symbol:     convert.SValOrEmpty(raw[1]),
-			Amount:     convert.F64ValOrZero(raw[2]),
-			AmountOrig: convert.F64ValOrZero(raw[3]),
-			Type:       convert.SValOrEmpty(raw[4]),
-			Status:     convert.SValOrEmpty(raw[5]),
-			Price:      convert.F64ValOrZero(raw[6]),
-			PriceAvg:   convert.F64ValOrZero(raw[7]),
-			MTSUpdated: convert.I64ValOrZero(raw[8]),
-		}
-		return
-	}
-
-	if len(raw) < 26 {
+	if len(raw) < 32 {
```

Comment:
should we use those length checks at all? when fields are updated via the api the client breaks, vs fields being not parsed...

Reply:
if api changes for one resource, maybe non crucial, program can still run vs whole thing panics and exits. Currently there's some duplication in pkg/models. Plan was to remove some old models afterwards and maybe reconsider checking length of payload. But that's for another PR as this one is ridiculous already 😄